src/stashcache.py

```diff
@@ -320,11 +320,6 @@ def generate_cache_grid_mapfile(global_data: GlobalData,
         if not resource:
             return ""
 
-    ligo_authz_list: List[AuthMethod] = []
```

Comment:
Why are you removing this?

Reply:
`generate_cache_grid_mapfile` is not using it, or am I wrong?

Reply:
Ah, you're right. That's fine, then.